lib/krikri/registry.rb

```diff
@@ -0,0 +1,45 @@
+module Krikri
+
+  ##
+  # Provide a registry of defined items that can be looked up by a token
```

Comment:
IOW, do we want to prefer symbols to strings, and do we need to do any string-to-symbol conversion here to assure appropriate resolution?